addons/website/views/snippets/s_product_list.xml

```diff
@@ -3,62 +3,95 @@
+                            </figure>
+                            <div class="card-body text-center">
+                                <h6 class="card-title">Subtle</h6>
+                            </div>
+                        </div>
                     </a>
-                    <div class="s_product_list_item_link">
-                        <a t-att-href="url" class="btn btn-secondary">Essential oils</a>
-                    </div>
                 </div>
                 <div class="col-lg-2 col-md-4 col-6">
-                    <a t-att-href="url">
-                        <img src="/web/image/website.s_product_list_default_image_5" alt="" class="img img-fluid"/>
+                    <a t-att-href="url" class="text-decoration-none">
+                        <div class="s_card o_card_img_top card o_cc o_cc1" data-name="Card" data-vcss="001">
+                            <figure class="o_card_img_wrapper ratio ratio-4x3 mb-0">
+                                <img src="/web/image/website.s_product_list_default_image_5" alt="" class="o_card_img card-img-top"/>
+                            </figure>
+                            <div class="card-body text-center">
+                                <h6 class="card-title">Sleek</h6>
+                            </div>
+                        </div>
                     </a>
-                    <div class="s_product_list_item_link">
-                        <a t-att-href="url" class="btn btn-secondary">Service</a>
-                    </div>
                 </div>
                 <div class="col-lg-2 col-md-4 col-6">
-                    <a t-att-href="url">
-                        <img src="/web/image/website.s_product_list_default_image_6" alt="" class="img img-fluid"/>
+                    <a t-att-href="url" class="text-decoration-none">
+                        <div class="s_card o_card_img_top card o_cc o_cc1" data-name="Card" data-vcss="001">
+                            <figure class="o_card_img_wrapper ratio ratio-4x3 mb-0">
+                                <img src="/web/image/website.s_product_list_default_image_6" alt="" class="o_card_img card-img-top"/>
+                            </figure>
+                            <div class="card-body text-center">
+                                <h6 class="card-title">Modern</h6>
+                            </div>
+                        </div>
                     </a>
-                    <div class="s_product_list_item_link">
-                        <a t-att-href="url" class="btn btn-secondary">Multimedia</a>
-                    </div>
                 </div>
             </div>
         </div>
     </section>
 </template>
 
+<record id="website.s_product_list_001_scss" model="ir.asset">
+    <field name="name">Product list 001 SCSS</field>
+    <field name="bundle">web.assets_frontend</field>
+    <field name="path">website/static/src/snippets/s_product_list/001.scss</field>
+</record>
+
```

Comment:
Nitpick: maybe move this just under or above the 000 SCSS asset, so they are grouped :slightly_smiling_face: